retis-events/src/events.rs

```diff
@@ -334,8 +343,8 @@ pub enum EventResult {
 /// having a proper structure is encouraged as it allows easier consumption at
 /// post-processing. Those objects can also define their own specialized
 /// helpers.
-pub trait EventSection: EventSectionInternal + for<'a> EventDisplay<'a> {}
-impl<T> EventSection for T where T: EventSectionInternal + for<'a> EventDisplay<'a> {}
+pub trait EventSection: EventSectionInternal + for<'a> EventDisplay<'a> + Send {}
+impl<T> EventSection for T where T: EventSectionInternal + for<'a> EventDisplay<'a> + Send {}
```

Comment:
And remove the `+ Send` requirement on `EventSection`s?
If anything changes in `Event` that makes it non-Sendable, isn't it better to have the automatic trait "inheritance" fail instead of having an explicit implementation that might mask the fact?

Reply:
> And remove the `+ Send` requirement on `EventSection`s? If anything changes in `Event` that makes it non-Sendable, isn't it better to have the automatic trait "inheritance" fail instead of having an explicit implementation that might mask the fact?

I was thinking about keeping the event section requirement too, just make the event one explicit. If that is explicit any addition to the type should make sure it doesn't break existing traits, but you have a good point that implicit implementation will actually make the build to fail (if events are used in a `Send` context).

The underlying question is: `retis-event` probably won't have a `Send` use of `Event`, how do we make sure the trait implicit implementation does not disappear? I guess if that is used in `retis` we should be aware pretty quickly :)

So, up to you, I guess that's fine as-is.